src/components/ChallengeModal.tsx/ChallengeModal.tsx

```diff
@@ -0,0 +1,41 @@
+import { Image, Modal, Typography } from "antd";
```

Comment:
I think the name of the folder has an extra `.tsx` in it 😅

Reply:
oh! that's true! thanks, Abhi! fixed: https://github.com/geo-quest/travel-bingo/pull/19/commits/b74380c8ed4ace0f676e4f14b379181d85c4fdba